### DISCUSSION TITLE
Is there a summary document on all the waveform approximants?

### DISCUSSION THREAD

**Discussion by yishu-xu** [gwastro/pycbc#4269] [Q&A]
Hello! Thanks for making this great project. 
I'm think if there exists a document that summarises all the available waveform approximants, with info such as does it include spin procession, tidal deformability, ring down etc., and with the references to their original papers? 

That will be extremely helpful!
Thanks, Shuxu

**Comment by ahnitz** [comment]
@yishu-xu This would be great to have, I agree. It doesn't exist at the moment, but we'd be happy to include it in the documentation if someone put it together or just wanted to get a start at it we could iterate on.

**Reply by Logggy** [reply]
Have there been any volunteers yet, or is the approximant document still in development limbo?

**Reply by ahnitz** [reply]
@Logggy If you know someone is able to volunteer or are able to contribute yourself that would be very much appreciated!

**Reply by spxiwh** [reply]
I don't really think that PyCBC is the right place to be documenting waveforms in lalsimulation. This should really live in lalsimulation ... But I couldn't find any useful information in lalsimulation's documentation pages (https://lscsoft.docs.ligo.org/lalsuite/lalsimulation/).

It would be good for us to have a top-level documentation page saying "We load waveforms from lalsimulation LINK FOR DETAILS and define a few of our own waveforms, or can load things through the interface Alex set up (maybe a list of things that support that) ... But as we don't provide much in the way of waveforms, our documentation on this point should be really be to point to the documentation of the code that does.

**Reply by ahnitz** [reply]
@spxiwh I think it makes sense to have a high level as you say, but since we support many sources of waveforms, it is definitely useful to have a single place to know where one can get which approximants and an idea why one might choose one approximant or the other. As you say, also with links to any available papers or documentation for the implementations.
